packages/server/graphql/public/typeDefs/Query.graphql

```diff
@@ -52,6 +52,21 @@ type Query {
 		isInvited: Boolean
 	): String
 
+	"""
+	Fetch the latest integration data for a poker estimate task
+	"""
+	refreshPokerEstimateIntegration(
```

Comment:
can we instead have the user query an object already on the schema? this looks like it's equivalent to `viewer { meeting(meetingId) { story(storyId) { ... }`